cirq-core/cirq/ops/clifford_gate.py

```diff
@@ -43,6 +43,36 @@ def _to_pauli_transform(matrix: np.ndarray) -> Optional[PauliTransform]:
     return None
 
 
+def _to_cliiford_tableau(
```

Comment:
```suggestion
def _to_clifford_tableau(
```

Reply:
Thanks for catching this. It is embarrassing to has this typo. (Relying too much on the auto-fill, lol). Fixed all other 12 places :)